kpops/pipeline.py

```diff
@@ -76,7 +78,7 @@ def add(self, component: PipelineComponent) -> None:
     def __bool__(self) -> bool:
         return bool(self.components)
 
-    def __iter__(self) -> Iterator[PipelineComponent]:
+    def __iter__(self) -> Iterator[PipelineComponent]:  # pyright: ignore [reportIncompatibleMethodOverride]
```

Comment:
do you know where this comes from? does Pydantic define this? I never had it in Pyright before